gym/core.py

```diff
@@ -60,11 +60,14 @@ def step(self, action):
     def reset(self):
         """Resets the state of the environment and returns an initial observation.
 
-        Returns: 
+        Returns:
             observation (object): the initial observation.
         """
         raise NotImplementedError
 
+    def get_pixels(self, *args, **kwargs):
```

Comment:
why is this necessary (vs using render directly)?

Reply:
See https://github.com/openai/gym/pull/1499#issuecomment-495817496.